recipes/ecodata/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "ecodata" %}
+{% set version = "0.2.2rc1" %}
```

Comment:
RC releases fall under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

You can either wait for a stable release of follow that document to get this package publish in a dev channel.

Reply:
We had to change our pyproject.toml to get the conda forge stuff to work, so we cut a rc build with just that to see if it would build. Since it is working, we pushed a stable release now since it is just a simple pyproject.toml change.

Reply:
These things are easier when you are the upstream author of the library :-)